spring-integration-redis/src/test/java/org/springframework/integration/redis/util/RedisLockRegistryTests.java

```diff
@@ -419,6 +421,59 @@ public void testEquals() throws Exception {
 		lock2.unlock();
```

Comment:
There is some point showing some inconsistency In this `testEquals()`:
```
lock1 = registry1.obtain("foo");
lock2 = registry2.obtain("foo");
assertNotEquals(lock1, lock2);
lock1.lock();
assertFalse(lock2.tryLock());
```
Since we can't lock `lock2`, they must be equal.

I'm developing now `HazelcastLockRegistry` and it shows me that lock from different registries for the same `key` are equal :smile: .